staging/src/k8s.io/client-go/tools/clientcmd/config.go

```diff
@@ -179,8 +182,13 @@ func ModifyConfig(configAccess ConfigAccess, newConfig clientcmdapi.Config, rela
 	}
 
 	startingConfig, err := configAccess.GetStartingConfig()
+
 	if err != nil {
-		return err
+		if isSetContext && errors.Is(err, ErrNoContextSet) {
```

Comment:
You need to be careful with `errors.Is()`. It returns true if at least one of the reported errors is the respective instance. However, please think about what should happen, if multiple errors are returned in the aggregate.

Reply:
My intent here was to keep the old behavior. Other errors should be reported as before. I added `isSetContext`, so one can still use `set-context`. I tried explaining this in the commit message.
Without this, the check I added caused an error on all the commands.
I will need some time to think of an alternative without `isSetContext`.

Reply:
Maybe I missed some detail why this is necessary. However, my main point is to discuss this change in an issue before investing more time into the implementation. It's possible that there is a reason for the current behavior that I don't know.

Reply:
I understand that the change should be discussed in an issue first. This is why I created one and initially sent a very minimal patch. 
Can the discussion for this issue happen on the original issue I opened in the kubectl repository or should I open a new issue in this repository?